### ISSUE TITLE
Modify data loader to adjust for class imbalance

### ISSUE THREAD

**Issue by PhilipMathieu** [PhilipMathieu/unet-orthoimagery#14]
Currently, the model overfits to images with no mask if trained on the "unbalanced" dataset, even when applying a weight in the BCELogitsLoss function.

Things we could try:
- "Downsample and Upweight" https://developers.google.com/machine-learning/data-prep/construct/sampling-splitting/imbalanced-data
- Augment only the true examples
- Other sampling techniques http://cs.uccs.edu/~jkalita/work/reu/REU2017/16Small.pdf

**Comment by PhilipMathieu** [comment]
Possible solution using a weighted sampler for the data loader: https://discuss.pytorch.org/t/how-to-handle-imbalanced-classes/11264
